### PR TITLE
feat: Add fuzzy parameter in search query

### PR DESCRIPTION
**Pull request by PedroHenrique-git**
## What's the purpose of this pull request?

Adds the possibility to pass the fuzzy parameter in the search query

## How to test it?

Change the parameter value between 0, 1 or auto and check the search result return

image
image

I used the carrefour account to test since that's where I found the problem, searching for the term 'Molico' was bringing up products like 'Molho de tomate'

### CODE REVIEW

**Comment by gvc** [comment]
Hey @PedroHenrique-git, thank you for your contribution. In your experience, would the value of `fuzzy` change for different pages or is it a store-wide configuration?

**Comment by PedroHenrique-git** [comment]
@gvc , I updated the pr

File: packages/api/src/platforms/vtex/index.ts [hunk 1]
```diff
@@ -92,16 +93,16 @@ const Resolvers = {
...
-      ctx.storage = {
-        channel: ChannelMarshal.parse(options.channel),
-        flags: options.flags ?? {},
-        locale: options.locale,
-      }
-      ctx.clients = getClients(options, ctx)
-      ctx.loaders = getLoaders(options, ctx)
-
-      return ctx
+  (ctx: any): Context => {
```
Comment (gvc) [inline packages/api/src/platforms/vtex/index.ts:96]: Formatting issue here?

**Review by gvc — CHANGES_REQUESTED** [review]

File: packages/api/src/platforms/vtex/index.ts (hunk 1 above)
Comment (PedroHenrique-git) [reply packages/api/src/platforms/vtex/index.ts:96]: yes, sometimes my vscode does this because of my configuration

Comment (PedroHenrique-git) [reply packages/api/src/platforms/vtex/index.ts:96]: @gvc this is a problem ?

**Comment by icazevedo** [comment]
The IS team communicated us that the fuzzy parameter should not be set by the users of the API. We're also using it wrong internally and will fix this behavior in the future. If you want to always enable or always disable the fuzzy functionality, you should open a support ticket requesting them to do that on their side.

With this context this PR is no longer relevant.

**Pull request closed without merging**